client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -269,9 +371,11 @@ export const PlanComparisonGrid: React.FC< PlanComparisonGridProps > = ( {
 			<Grid>
 				<PlanComparisonGridHeader
 					displayedPlansProperties={ displayedPlansProperties }
+					visiblePlansProperties={ visiblePlansProperties }
```

Comment:
Needed to pass both to:
* respect the order of the visible plans via `visiblePlansProperties`
* determine which plans to mention in the plan dropdowns via `displayedPlansProperties`